src/components/VisualizationTypeSelector/styles/VisualizationTypeSelector.module.css

```diff
@@ -3,17 +3,29 @@
     margin-left: auto;
 }
 
+.arrowIcon.listIsOpen {
+    transform: rotate(180deg);
+}
+
 .button {
     padding: 7px var(--spacers-dp8);
     display: flex;
     align-items: center;
     justify-content: flex-start;
     background-color: var(--colors-white);
     cursor: pointer;
+    flex-grow: 1
+}
+
+.button:hover,
+.button.listIsOpen {
+    background-color: var(--colors-grey200);
 }
 
-.button > span {
+.button > .selectedVizTypeLabel {
     font-size: 14px;
+    padding-top: 1px;
```

Comment:
This padding is problematic. I'm not actually sure if it adds any value moving the text down here. In the LL I did a similar fix, which does make sense because we always show the same icon. However, in DV each viz-type comes with its own icon. In some cases things now look better aligned, but for other icons this is not the case.